src/Kafka/Kafka.purs

```diff
@@ -0,0 +1,133 @@
+module Kafka.Kafka
+  ( LogLevel(..)
+  , Kafka
+  , KafkaConfig
+  , KafkaConfigBrokers(..)
+  , newKafka
+  ) where
+
+import Prelude
+
+import Control.Promise as Control.Promise
+import Data.Maybe as Data.Maybe
+import Effect as Effect
+import Effect.Aff as Effect.Aff
+import Effect.Uncurried as Effect.Uncurried
+import Kafka.FFI as Kafka.FFI
+import Untagged.Union as Untagged.Union
+
+-- | https://github.com/tulios/kafkajs/blob/dcee6971c4a739ebb02d9279f68155e3945c50f7/types/index.d.ts#L17
```

Comment:
Can we add some documentation to all these types and functions instead of just pointing to the wrapped library?

Reply:
I'll add documentation to our public API. These types for FFI are directly translated from TypeScript definitions so I cited them here to make errors easier to spot.

Reply:
Gotcha!

Reply:
@arthurxavierx I've added new comments to public API types. Do you want to take a look?

Reply:
Looks good!